meinberlin/apps/plans/serializers.py

```diff
@@ -7,95 +7,72 @@
+            "point",
+            "point_label",
             "published_projects_count",
-            "created_or_modified",
+            "status",
+            "subtype",
             "tile_image",
             "tile_image_alt_text",
             "tile_image_copyright",
+            "title",
+            "topics",
+            "type",
+            "url",
         ]
 
-    def get_subtype(self, instance):
-        return "plan"
-
-    def _get_participation_status_plan(self, item):
-        return item.get_status_display(), not bool(item.status)
-
-    def get_type(self, instance):
-        return "plan"
+    def get_url(self, plan: Plan):
+        return plan.get_absolute_url()
 
-    def get_url(self, instance):
-        return instance.get_absolute_url()
+    def get_published_projects_count(self, plan: Plan) -> int:
+        return plan.published_projects_count
 
-    def get_published_projects_count(self, instance):
-        return instance.published_projects.count()
+    def get_participation_string(self, plan: Plan):
+        return plan.get_status_display()
 
-    def get_participation_string(self, instance):
-        (
-            participation_string,
-            participation_active,
-        ) = self._get_participation_status_plan(instance)
-        return str(participation_string)
+    def get_participation_active(self, plan: Plan):
+        return not bool(plan.status)
 
-    def get_participation_active(self, instance):
-        (
-            participation_string,
-            participation_active,
-        ) = self._get_participation_status_plan(instance)
-        return participation_active
-
-    def get_tile_image(self, instance):
-        image_url = ""
-        if instance.tile_image:
-            image = get_thumbnailer(instance.tile_image)["project_tile"]
-            image_url = image.url
-        elif instance.image:
-            image = get_thumbnailer(instance.image)["project_tile"]
-            image_url = image.url
-        return image_url
-
-    def get_tile_image_alt_text(self, instance):
-        if instance.tile_image:
-            return instance.tile_image_alt_text
-        elif instance.image_alt_text:
-            return instance.image_alt_text
+    def get_tile_image(self, plan: Plan):
+        if plan.tile_image:
+            return get_thumbnailer(plan.tile_image)["project_tile"].url
+        elif plan.image:
+            return get_thumbnailer(plan.image)["project_tile"].url
         else:
-            return None
+            return ""
 
-    def get_tile_image_copyright(self, instance):
-        if instance.tile_image:
-            return instance.tile_image_copyright
-        elif instance.image_copyright:
-            return instance.image_copyright
-        else:
-            return None
+    def get_tile_image_alt_text(self, plan: Plan):
+        if plan.tile_image:
+            return plan.tile_image_alt_text
+        elif plan.image_alt_text:
+            return plan.image_alt_text
```

Comment:
reading this function makes it quite easy to think there are only two cases (e.g. tile_image or image_alt_text), however returning None is also possible. While it's implicitly there I find it easier to read with the explicit `else`. Or at least have an `Optional` type hint so it's clear?